frontends/precinct-scanner/src/screens/poll_worker_screen.tsx

```diff
@@ -271,11 +253,61 @@ export function PollWorkerScreen({
     }
   }
 
+  const [pollWorkerFlowState, setPollWorkerFlowState] = useState<
+    PollWorkerFlowState | undefined
+  >(
+    isPollsOpen
+      ? PollWorkerFlowState.CLOSE_POLLS_FLOW__CONFIRM
+      : PollWorkerFlowState.OPEN_POLLS_FLOW__CONFIRM
+  );
+  function showAllPollWorkerActions() {
+    return setPollWorkerFlowState(undefined);
+  }
+
   async function printTallyReport(copies: number) {
-    await printer.print({
-      sides: 'one-sided',
-      copies,
-    });
+    if (!electionDefinition) return;
+    if (!precinctSelection) return;
+    if (!currentCompressedTally) return;
+    if (!currentSubTallies) return;
```

Comment:
Do we want to just return and do nothing in these cases? Or are these more meant to be assertions?

Reply:
Based on the logic of the page, these could be assertions. Are meant to be assertions. I don't really understand the tradeoffs in these situations. It seems aggressive to assert things and risk throwing an error but... these should always exist. Should I just make them assertions?

Reply:
I like to use assertions for clarity to the reader. The question of "what happens if an assertion fails in production" is one we can address separately (e.g. we can handle the error gracefully). I'm not totally sure what happens if an assertion fails today, but in general loud failures are easier to deal with than quiet ones.